crates/eval/src/lib.rs

```diff
@@ -27,15 +27,37 @@ struct EvalArgs {
     #[arg(long)]
     pub shard_size: Option<usize>,
 
-    /// The Slack channel ID to post results to, only required if you want to post to Slack.
+    /// Whether to post results to Slack, defaults to false.
+    #[clap(
+        long,
+        default_missing_value("true"),
+        default_value("false"),
+        num_args(0..=1),
+        require_equals(true),
+        action = ArgAction::Set,
+    )]
+    pub post_to_slack: bool,
+
+    /// The Slack channel ID to post results to, only used if post_to_slack is true.
     #[arg(long)]
     pub slack_channel_id: Option<String>,
 
-    /// The Slack bot token to post results to, only used if slack_channel_id is set.
+    /// The Slack bot token to post results to, only used if post_to_slack is true.
     #[arg(long)]
     pub slack_token: Option<String>,
 
-    /// The GitHub token for authentication, only required if you want to post to GitHub.
+    /// Whether to post results to GitHub PR, defaults to false.
+    #[clap(
```

Comment:
if you're wondering why we have to do this madness, see https://github.com/clap-rs/clap/issues/1649#issuecomment-1837123432